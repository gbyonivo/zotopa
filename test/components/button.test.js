import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../../src/components/button';

describe('Button component', () => {
  it('renders correctly when isLoading is true', () => {
    const actual = renderer.create(
      <Button
        value="play"
        onClick={() => {}}
        isLoading={true}
      />
    );
    expect(actual).toMatchSnapshot();
  });
});

describe('Button component', () => {
  it('renders correctly when isLoading is false', () => {
    const actual = renderer.create(
      <Button
        value="play"
        onClick={() => {}}
      />
    );
    expect(actual).toMatchSnapshot();
  });
});