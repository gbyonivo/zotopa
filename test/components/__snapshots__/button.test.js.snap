// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders correctly when isLoading is false 1`] = `
<div
  className="buttonDiv"
>
  <button
    onClick={[Function]}
  >
    play
  </button>
</div>
`;

exports[`Button component renders correctly when isLoading is true 1`] = `
<div
  className="buttonDiv"
>
  <button
    disabled={true}
    onClick={[Function]}
  >
    play...
  </button>
</div>
`;
